assets/scss/components/_attorney-banner.scss

```diff
@@ -72,3 +72,19 @@ Styleguide Attorney Banner
   font-size: 19px;
   margin: 0;
 }
+
+//to override background-colour and border-left of attorneyBanner
```

Comment:
please move the `.attorneyBanner--*`work up to sit after the `.attorneyBanner` work